modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableRowSkinBase.java

```diff
@@ -349,12 +349,13 @@ protected ObjectProperty<Node> graphicProperty() {
                 height = h;
             }
 
-            width = tableCell.prefWidth(height);
-
             if (isVisible) {
                 if (fixedCellSizeEnabled && tableCell.getParent() == null) {
                     getChildren().add(tableCell);
                 }
+                // Note: prefWidth() has to be called only after the tableCell is added to the tableRow, if it wasn't
```

Comment:
Minor: there is no explicit rule about limiting line comments to 80 characters, as far as I know, but it might be good to apply it to your comments (also to the test ones)

Reply:
Ok, right, I missed that.